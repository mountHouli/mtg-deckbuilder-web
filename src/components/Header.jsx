import React from 'react'

const Header = () => {
  return (
    <div className="header-container">
      MTG Deckbuilder
    </div>
  )
}

export default Header
