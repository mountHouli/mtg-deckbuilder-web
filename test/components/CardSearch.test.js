import React from 'react'
import renderer from 'react-test-renderer'

import Header from 'components/Header'

it('Header snapshot matches', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
})
