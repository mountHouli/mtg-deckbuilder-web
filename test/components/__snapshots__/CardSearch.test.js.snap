// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSearch Snapshot Matches 1`] = `
<div
  className="header-container"
>
  MTG Deckbuilder
</div>
`;
